### DISCUSSION TITLE
Wrong usage for group license

### DISCUSSION THREAD

**Discussion by Weinwurm** [dashingsoft/pyarmor#1312] [Q&A] [answered]
After trying to use the enterprise license with
`pyarmor -d reg .\pyarmor-regfile-6029.zip`
I get a 
`ERROR    wrong usage for group license, please check pyarmor reg in Man page`

**Comment by jondy** [comment]
Please check online documentation about how to use group license.

**Reply by Weinwurm** [reply]
Please provide a decent documentation. We are paying for this. Thanks

**Reply by jondy** [reply]
Please check activation file `pyarmor-regcode-xxxx.txt`, there is guide how to use group license and doc link

**Reply by Weinwurm** [reply]
The guide is wrong. 
So what? Could be please be more helpful to paying customers? 
And the entire documentation is a incredible mess. Especially regarding group licenses. 
Thanks!

**Comment by 1998jack1998** [comment]
I have bought group license, but most of may work I do using dev docker containers in wsl so every time I made some change in container I need to use license for another device is there any possibility to omit this limitation? Because 100 devices when I use dev container means almost nothing.

**Comment by jondy** [comment]
@1998jack1998 group license will support unlimited dockers which are not highly customized in this month.

**Reply by 1998jack1998** [reply]
@jondy nice to hear that. Thank you for your fast reply :D

**Comment by jondy** [comment, marked as answer]
Please refer to
https://pyarmor.readthedocs.io/en/stable/how-to/register.html
